processor/k8sattributesprocessor/factory.go

```diff
@@ -141,13 +141,20 @@ func createKubernetesProcessor(
 	options ...option,
 ) (*kubernetesprocessor, error) {
 	kp := &kubernetesprocessor{logger: params.Logger}
+	pCfg := cfg.(*Config)
 
-	err := errWrongKeyConfig(cfg)
+	err := errWrongKeyConfig(pCfg)
 	if err != nil {
 		return nil, err
 	}
 
-	allOptions := append(createProcessorOpts(cfg), options...)
+	if len(pCfg.Extract.Metadata) == 0 {
+		kp.logger.Warn("Default metadata extraction rules will be changed in future releases, " +
+			"please specify the extraction rules explicitly in `processors::k8sattributes::extract::metadata` field. " +
```

Comment:
```suggestion
			"please specify the extraction rules explicitly in the `processors::k8sattributes::extract::metadata` field. " +
```
Just a simple wording change